deployer/scripts/hawkular.sh

```diff
@@ -208,6 +208,17 @@ EOF
   fi
   # this may return an error code if the route already exists, this is to be expect with a refresh and is why we have the || true here
   oc create route reencrypt hawkular-metrics ${route_params} || true
+
+  # if we are dealing with a refresh, then we need to update the router's CACertificate for the Hawkular Metric's internal certificate
+  if [ "$mode" = "refresh" ]; then
+   type=$(oc get route hawkular-metrics --template='{{.spec.tls.termination}}') || true
+   if [ "$type" = "reencrypt" ]; then
+     #todo: there is probably a much more elegrant way to do this.
```

Comment:
yeah, I had to merge it quickly since the refresh option was broken without this change.

Can you please open a new PR to fix this in a more elegant manner?